core/src/main/scala/org/apache/spark/sql/delta/commands/cdc/CDCReader.scala

```diff
@@ -386,6 +386,10 @@ trait CDCReaderImpl extends DeltaLogging {
       throw DeltaErrors.changeDataNotRecordedException(start, start, end)
     }
 
+    if(startVersionSnapshot.protocol.isFeatureSupported(DeletionVectorsTableFeature)) {
```

Comment:
can we use `DeletionVectorUtils.deletionVectorsReadable`? 

nit: space after `if`